api/db/setup_local_database.sh

```diff
@@ -41,7 +39,7 @@ trap finish EXIT
 cat drop_db.sql create_db.sql | envsubst > $CREATE_DB_FILE
 
 echo "Creating empty database..."
-mysql -u "$ROOT_DB_USER" $ROOT_PASSWORD_ARGS < ${CREATE_DB_FILE}
+mysql -u "$ROOT_DB_USER" -p${ROOT_PASSWORD} < ${CREATE_DB_FILE}
```

Comment:
What's the error on Macs / what does this fix?

It looks like this will break if you use --nopassword, since `-p` prompts interactively for a password. No?

We should prefer converting scripts to Python when they are broken, as opposed to investing in bash scripts (for portability, error reporting, reusability long term). Is this an opportunity to rewrite this one? (Dan, I think you called it out as a TODO to make this a Python script post-demo?)